File: frontend/src/app/core/model/customer-review-response.ts
export interface PrimaryContactResponse {
  id: number;
  firstName: string;
  lastName: string;
  email: string;
  phoneNumber: string | null;
  designation: string | null;
}

export interface CustomerReviewResponse {
  id: number;
  companyName: string;
  taxId: string;
  companyEmail: string;
  phoneNumber: string | null;
  businessAddress: string | null;
  industryType: string | null;
  status: string;
  rejectionReason: string | null;
  primaryContact: PrimaryContactResponse | null;
}

File: src/main/java/com/cblos/dto/CustomerReviewResponse.java
package com.cblos.dto;

public record CustomerReviewResponse(
        Integer id,
        String companyName,
        String taxId,
        String companyEmail,
        String phoneNumber,
        String businessAddress,
        String industryType,
        String status,
        String rejectionReason,
        PrimaryContactResponse primaryContact) {
}